calc/calc.py

```diff
@@ -79,7 +79,7 @@
 }
 
 SUFFIXES = {
```

Comment:
`SUFFIXES` is used in one other place. I'm surprised it didn't break any tests.

Let's make sure we don't break existing logic.

You could leave this dict as it was originally and add a second dict, for example:
```
SUFFIX_EVALS = {
    '%': lambda x: x * SUFFIXES['%']
}
```

Reply:
I looked at the other uses of `SUFFIXES` and it should be fine. The other uses are only checking the key of the object to find what should be considered a suffix and not using the value in the object to evaluate anything.